tests/network-ovn

```diff
@@ -2127,6 +2127,55 @@ ovn_leases_tests() {
         lxc network delete lxdbr0
 }
 
+ovn_vlan_uplink_tests() {
+    echo "==> vlan uplink"
+
+    sudo ovs-vsctl set open_vswitch . \
+        external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+        external_ids:ovn-encap-type=geneve \
+        external_ids:ovn-encap-ip=127.0.0.1
+
+    lxc network create lxdbr0 \
+        ipv4.address=10.10.10.1/24 ipv4.nat=true \
+        ipv4.dhcp.ranges=10.10.10.2-10.10.10.199 \
+        ipv6.address=fd42:4242:4242:1010::1/64 ipv6.nat=true \
+        ipv6.ovn.ranges=fd42:4242:4242:1010::200-fd42:4242:4242:1010::254
+
+    echo "==> Create physical UPLINK network with VLAN config set to 10"
+    lxc network create UPLINK --type=physical parent=lxdbr0 \
+        ipv4.ovn.ranges=10.10.10.200-10.10.10.254 \
+        ipv4.gateway=10.10.10.1/24 \
+        vlan=10
+
+    echo "==> Create OVN network"
+    lxc network create ovn-virtual-network --type=ovn network=UPLINK
+    sleep 2
+
+    echo "==> Set up containers"
+    lxc launch ubuntu:24.04 c1-ovn -n ovn-virtual-network
+    lxc launch ubuntu:24.04 c1-lxdbr0 -n lxdbr0
+    bridge vlan add dev lxdbr0 vid 10 tagged self
```

Comment:
Does that allow VLAN tagged packets/frames to be accepted by `lxdbr0` rather than being discarded? If so, a comment to that effect would help (me at least ;) ).